src/helpers/generateReadTime.js

```diff
@@ -0,0 +1,21 @@
+/*
+Estimation of read time inspired by Nick Fisher
+Link: https://blog.medium.com/read-time-and-you-bc2048ab620c
+*/
+const generateReadTime = (text) => {
+  // number of images in the article
+  const countImages = (text.match(/<img /g) || []).length;
+  // image seconds
+  let imageSeconds = 0;
+  if (countImages === 1) imageSeconds = 12;
+  if (countImages === 2) imageSeconds = 22;
+  if (countImages > 2) imageSeconds = 22 + [countImages * 2];
+  // number of words
+  const numberOfWords = text.split(' ').length;
+  // calculate seconds you can read the text
+  const wordsSeconds = (numberOfWords * 60) / 265;
```

Comment:
the values you have in this file should be defined as constants and reused where appropriate